test/response_to_json_translator_test.cc

```diff
@@ -88,7 +88,7 @@ class ResponseToJsonTranslatorTestRun {
     // While we still have expected messages before or at the current position
     // try to match.
     while (next_expected_ != std::end(expected_) &&
-           next_expected_->at <= position_) {
+        next_expected_->at <= position_) {
```

Comment:
Please fix the coding format

Reply:
This seems to edit lots of files we have right now, so I will open another PR just for the formatting. 
Removing this format change from this PR for now.

Reply:
Sorry, I just re-ran `script/check-style` and it didn't modify any files. I don't know why I was seeing multiple files edited earlier.
But we are good for now.